src/components/profile/ProfileDataTable.js

```diff
@@ -40,87 +39,85 @@ const ProfileDataTable = ({
 
   return (
     <Section.Row alignItems="center" grow={1}>
-      {/* <KeyboardAwareScrollView resetScrollToCoords={{ x: 0, y: 0 }} scrollEnabled={false}> */}
-      <Section.Row>
-        <InputRounded
-          disabled={!editable}
-          error={errors.username}
-          icon="username"
-          iconColor={theme.colors.primary}
-          iconSize={22}
-          onChange={username => onChange({ ...profile, username })}
-          placeholder="Choose a Username"
-          value={profile.username}
-        />
-      </Section.Row>
-      <Section.Row>
-        {editable ? (
-          <Section.Stack grow>
-            <Section.Row>
-              <PhoneInput
-                error={errors.mobile && errors.mobile !== ''}
-                id="signup_phone"
-                onFocus={() => setLockSubmit(true)}
-                onChange={value => onChange({ ...profile, mobile: value })}
-                onBlur={() => {
-                  setLockSubmit(false)
-                  verifyPhone()
-                }}
-                placeholder="Enter phone number"
-                value={profile.mobile}
-                style={{
-                  borderColor: errors.mobile ? theme.colors.red : theme.colors.lightGray,
-                  borderRadius: 24,
-                  borderWidth: 1,
-                  color: errors.mobile ? theme.colors.red : theme.colors.text,
-                  paddingBottom: 0,
-                  paddingLeft: 0,
-                  paddingRight: 0,
-                  paddingTop: 0,
-                  position: 'relative',
-                }}
-              />
-              <Section.Row style={styles.suffixIcon}>
-                <Icon
-                  color={errors.mobile ? theme.colors.red : theme.colors.primary}
-                  name="phone"
-                  size={28}
-                  style={styles.phoneIcon}
+      <KeyboardAwareScrollView resetScrollToCoords={{ x: 0, y: 0 }} scrollEnabled={false}>
+        <Section.Row>
+          <InputRounded
+            disabled={!editable}
+            error={errors.username}
+            icon="username"
+            iconColor={theme.colors.primary}
+            iconSize={22}
+            onChange={username => onChange({ ...profile, username })}
+            placeholder="Choose a Username"
+            value={profile.username}
+          />
+        </Section.Row>
+        <Section.Row>
+          {editable ? (
+            <Section.Stack grow>
+              <Section.Row>
+                <EditPhoneInput
+                  error={errors.mobile && errors.mobile !== ''}
```

Comment:
how will errors be shown on native? do they look the same? are they shown at all?

Reply:
lots of these props are not used in editphonenumber.native which means behaviour will not be the same on native

Reply:
Looks good
![Screenshot_2020-02-05-11-48-32-042_com gooddollar](https://user-images.githubusercontent.com/59056508/73852551-22465c80-480e-11ea-812f-ae539810f94c.jpg)
![Screenshot_2020-02-05-11-48-51-036_com gooddollar](https://user-images.githubusercontent.com/59056508/73852556-24102000-480e-11ea-9796-2471369ea3e1.jpg)